packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -78,7 +94,17 @@ export class LicenseManager {
 		} catch (e) {
 			throw new Error('Could not parse object')
 		}
-		return licenseInfoValidator.validate(decodedData)
+		try {
+			return licenseInfoValidator.validate(decodedData)
+		} catch (e: any) {
+			if (e.message.includes('Unexpected property')) {
+				this.outputMessages([
+					'License key contains some unknown properties.',
+					'You may want to update tldraw packages to a newer version to get access to new functionality.',
+				])
+			}
+		}
```

Comment:
we should log an error to sentry if the validation fails for other reasons (other than an unknown property)

Reply:
How should that work? We now only use sentry for our private stuff (dotcom,...). Did you think we'd set up a separate sentry project for it? How would we handle secrets?

Reply:
Oh, right right. Nevermind, yes you're right.